src/EnergyPlus/WaterToAirHeatPumpSimple.cc

```diff
@@ -3888,7 +3888,8 @@ namespace WaterToAirHeatPumpSimple {
             state.dataWaterToAirHeatPumpSimple->GetCoilsInputFlag = false;
         }
 
-        if (CoilType == "COIL:COOLING:WATERTOAIRHEATPUMP:EQUATIONFIT" || CoilType == "COIL:HEATING:WATERTOAIRHEATPUMP:EQUATIONFIT") {
+        if (UtilityRoutines::SameString(CoilType, "COIL:COOLING:WATERTOAIRHEATPUMP:EQUATIONFIT") ||
```

Comment:
@Myoldmopar OK. I am going to pause for the time being.

Reply:
I would say leave this for now, but open a new refactoring task to clean up both [``WaterToAirHeatPumpSimple::GetCoilAirFlowRate](https://github.com/NREL/EnergyPlus/blob/900b824d35cdd34ef500cf0eb7391408534fc0a9/src/EnergyPlus/WaterToAirHeatPumpSimple.cc#L3863C4-L3863C4) and [`VariableSpeedCoils::GetCoilAirFlowRateVariableSpeed`](https://github.com/NREL/EnergyPlus/blob/900b824d35cdd34ef500cf0eb7391408534fc0a9/src/EnergyPlus/VariableSpeedCoils.cc#L7868).

a. The coil type strings should be enums. The calling functions should already know the coil type enum by the time this is called.
b. The coil name can be replaced with coil index. Again, the calling functions should already know this.
c. I think the check for getinput can be removed from these and just test for a valid coil index and a valid type enum and throw an error.
@rraustad Would you agree?

Reply:
Yes I agree. DataHVACGlobals has a CoilType enum and coilTypeNamesUC array that can be used for that. If the coil name is needed for something it can be looked up. Regarding getting rid of getInput in these mining functions, I'm not sure of that at this point. The calling function would need to use the index on this call and that might be a little more work to cover all parent objects. We are definitily moving in this direction.

Reply:
Good, I don't even have to make these types of comments anymore! I can move on to better and more pedantic comments about the use of `this->m_X`.

Reply:
@mjwitte @rraustad @amirroth I leave it as is for future refactory.